agent/functional_tests/util/utils_unix.go

```diff
@@ -150,6 +151,11 @@ func (agent *TestAgent) StartAgent() error {
 		Cmd: strings.Split(os.Getenv("ECS_FTEST_AGENT_ARGS"), " "),
 	}
 
+	// Append ECS_INSTANCE_ATTRIBUTES to dockerConfig
+	if attr := os.Getenv("ECS_INSTANCE_ATTRIBUTES"); attr != "" {
```

Comment:
Reverted back to the old style to prevent code-duplication and run the test on both platforms